Tests/SSL/test_dataset/Data_Entry_2017.csv

```diff
@@ -0,0 +1,301 @@
+﻿Image Index
+1.dcm
+1.dcm
```

Comment:
Is there no better way of storing/creating this file? it is, well, rather repetitive?

Reply:
switched to on the fly creation